apps/src/JSInterpreter.js

```diff
@@ -372,13 +372,17 @@ JSInterpreter.prototype.executeInterpreter = function (firstStep, runUntilCallba
     if ((reachedBreak && !unwindingAfterStep) ||
         (doneUserLine && !unwindingAfterStep && !atMaxSpeed) ||
         this.yieldExecution ||
+        this.interpreter.paused_ ||
```

Comment:
Can `this.yieldExecution` be removed, and `JSInterpreter.prototype.yield` implemented with `this.interpreter.paused_`?